### PR TITLE
History

### PR DESCRIPTION
**Pull request by CalvinDSeamons**
Addresses issue #45 
Pavilion status flag added ```--history $test_id```
Full command looks like: `pav status --history 123` 

Right now the flag just takes a singular test_id and returns a output table with all the state history associated with that id.  Currently looks like:
```bash
 Test state history                                                                                                     
---------+----------------+---------------+----------------------------+----------------------------------------------
 Test id | Name           | State         | Time                       | Note                                         
---------+----------------+---------------+----------------------------+----------------------------------------------
 333     | something2.two | CREATED       | 2020-04-20T11:27:56.003436 | Created status file.                         
 333     | something2.two | CREATED       | 2020-04-20T11:27:56.005275 | Test directory and status file created.      
 333     | something2.two | BUILD_CREATED | 2020-04-20T11:27:56.011967 | Builder created.                             
 333     | something2.two | CREATED       | 2020-04-20T11:27:56.019893 | Test directory setup complete.               
 333     | something2.two | BUILD_REUSED  | 2020-04-20T11:27:56.076920 | Test 8514a134a36f1d25 run 333 reusing build. 
 333     | something2.two | SCHEDULED     | 2020-04-20T11:27:56.497877 | Test raw has job ID fg-fey1.lanl.gov_20528.  
 333     | something2.two | PREPPING_RUN  | 2020-04-20T11:27:57.264974 | Converting run template into run script.     
 333     | something2.two | RUNNING       | 2020-04-20T11:27:57.267015 | Starting the run script.                     
 333     | something2.two | RUNNING       | 2020-04-20T11:27:57.270641 | Currently running.                           
 333     | something2.two | RUN_DONE      | 2020-04-20T11:27:57.279072 | Test run has completed.                      
 333     | something2.two | RESULTS       | 2020-04-20T11:27:57.281502 | Parsing 5 result types.                      
 333     | something2.two | COMPLETE      | 2020-04-20T11:27:57.285130 | The test completed with result: PASS  
```
Open to stylistic changes of the table. I find `name, test_id` redundant, also time could definitely be formatted in nicer way if need be. I wasn't to sure on what data should be present.

### CODE REVIEW

**Comment by Paul-Ferrell** [comment]
Run the datetimes through `output.get_relative_timestamp`. If you set that function as the 'field_info' property for that field in draw_table, the table will handle things on the fly.

File: lib/pavilion/plugins/commands/status.py [hunk 1]
```diff
@@ -171,6 +172,54 @@ def get_statuses(pav_cfg, args, errfile):
...
+                    'name': name_final,
+                    'state': val[1],
+                    'time': val[0],
+                    'note': val[2]
+                })
+    except (TestRunError, TestRunNotFoundError) as err:
+        output.fprint("The test_id {} does not exist in your "
+                      "working directory.".format(args.history),
+                      color=output.RED)
+        ret_val = 0
```
Comment (Paul-Ferrell) [inline lib/pavilion/plugins/commands/status.py:209]: errno.EINVAL

**Comment by Paul-Ferrell** [comment]
Drop the test id and name columns, and add that info to the table title.

**Comment by Paul-Ferrell** [comment]
You still need to run the times through the formatter.

**Comment by CalvinDSeamons** [comment]
> You still need to run the times through the formatter.

**Comment by CalvinDSeamons** [comment]
History output now looks like: 
Screen Shot 2020-05-27 at 11 08 02 AM

Also added a unit test to make sure bad arguments are caught correctly.

**Pull request merged**